internal/testutils/fixtures.go

```diff
@@ -27,9 +29,22 @@ var TestApplicationData = []m.Application{
 	{ID: 2, Extra: datatypes.JSON("{\"extra\": false}"), ApplicationTypeID: 1, SourceID: 1, TenantID: 1},
 }
 
+var (
+	CreatedAt, _ = time.Parse("2006-01-02T15:04:05.999Z", "2021-11-26 18:47:19 CET")
+	UpdatedAt    = CreatedAt
+	port         = 80
+	defaultValue = true
```

Comment:
Here I'm noticing that both of the endpoints are created as "default". It seems that the table allows it, although we don't allow it when validating a new endpoint creation. Maybe we should set one of them to `false` and the other one to `true`? What do you think?

Reply:
that's right and it was fixed 👍